.travis.yml

```diff
@@ -5,4 +5,4 @@ node_js:
 env:
 - TEST_DIR=.
 - TEST_DIR=scripts/babel-relay-plugin
-script: cd $TEST_DIR && npm install && npm test
+script: cd $TEST_DIR && npm install && npm run lint && npm test
```

Comment:
ok so to satisfy 1. I can just remove `npm run lint` from travis config

and then submit another PR with the proper config?